javascript/organizing_your_javascript_code/objects_and_object_constructors.md

```diff
@@ -244,7 +244,9 @@ If we had used `Object.create` in this example, then we could safely edit the `N
 
 1. Read up on the concept of the prototype from the articles below. 
    1. [JavaScriptIsSexy's article on the prototype](https://web.archive.org/web/20200513181548/https://javascriptissexy.com/javascript-prototype-in-plain-detailed-language/) is a straightforward introduction and demonstration of the concept. It also covers constructors again.. good time for a review! The important bits here, once you've covered the basics, are 'Prototype-based inheritance' and the 'Prototype chain'.
-   2. To go a bit deeper into both the chain and inheritance, spend some time with [JavaScript.Info's article on Prototypal Inheritance](http://javascript.info/prototype-inheritance). As usual, doing the exercises at the end will help cement this knowledge in your mind. Don't skip them! Important note: This article makes heavy use of `__proto__` which is not generally recommended. The concepts here are what we're looking for at the moment. We will soon learn another method or two for setting the prototype.
+   2. [This video](https://www.youtube.com/watch?v=4jb4AYEyhRc) is a good overview of `Prototype` & [This video](https://www.youtube.com/watch?v=Fsp42zUNJYU) is a good overview of `Prototype Inheritance` by The Net Ninja.
+   3. To go a bit deeper into both the chain and inheritance, spend some time with [JavaScript.Info's article on Prototypal Inheritance](http://javascript.info/prototype-inheritance). As usual, doing the exercises at the end will help cement this knowledge in your mind. Don't skip them! Important note: This article makes heavy use of `__proto__` which is not generally recommended. The concepts here are what we're looking for at the moment. We will soon learn another method or two for setting the prototype.
+   4. 
```

Comment:
You need to fix the formatting issue with the 4th list entry on line 249 that is empty

Reply:
fixed!